packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -26,13 +26,20 @@ const AppContents = ({
     platform,
     pageView,
     pushDataLayer,
+    setAppContentsScrollRef,
 }) => {
     const [show_cookie_banner, setShowCookieBanner] = React.useState(false);
     const [is_gtm_tracking, setIsGtmTracking] = React.useState(false);
     const { is_appstore } = React.useContext(PlatformContext);
 
     const tracking_status = tracking_status_cookie.get(TRACKING_STATUS_KEY);
 
+    const scroll_ref = React.useRef(null);
+
+    React.useEffect(() => {
+        if (scroll_ref) setAppContentsScrollRef(scroll_ref);
+    }, []);
```

Comment:
@farzin-fs this scroll_ref is redundant in this useEffect dependency because we mount app-contents only once on the first loading of the app and all our rerenders happen inside this component